docs/terms.md

```diff
@@ -0,0 +1,30 @@
+# Terms and Conditions
+
+Clangd remote index service is run on a best-effort basis by
+c-tools-team@google.com. This project is sponsored by Google but is not an
+official Google product. We also monitor the service on a best-effort basis and
```

Comment:
I would tweak the wording a bit to something like:
"Google donated the source code and cloud resources to the LLVM project, but is not an official Google product."